packages/fennel-ls/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: fennel-ls
+description: Language Server for Fennel
+homepage: https://git.sr.ht/~xerool/fennel-ls
+licenses:
+  - MIT
+languages:
+  - Fennel
+categories:
+  - LSP
+
+source:
+  id: pkg:luarocks/fennel-ls@0.1.1-1
```

I see there's a new version since the PR was opened, let's update right away.

```suggestion
  id: pkg:luarocks/fennel-ls@0.1.2-2
```

Unfortunately Renovate (which is used to automate version upgrades) does not support Luarocks so this will have to manually updated in the future.